test/quizzes_test.py

```diff
@@ -1,20 +1,63 @@
 import unittest
-
 from app.controllers.quizzes_controller import QuizzesController
+from datetime import datetime
 
 class QuizzesTest(unittest.TestCase):
 
     def setUp(self):
         # Run tests on non-production data
-        self.ctrl = QuizzesController('quizzes_test.py')
-        
+        self.ctrl = QuizzesController('quizzes_test.json')
+
     def test_expose_failure_01(self):
         """
-        Implement this function and two more that
-        execute the code and make it fail.
+        This test case aims to expose a crash when adding a quiz with missing parameters.
+        Expected crash: quizzes_controller.py, line 64
         """
-        self.assertTrue(True, 'Example assertion.')
+
+        self.ctrl.clear_data()
+        # Not passing the available_date param
+        # This will give missing positional argument: 'available_date'
+        quiz_id = self.ctrl.add_quiz(title="Quiz title",text="text", due_date=datetime.now())
+        # Check that we have one quiz in the list
+        quizzes = self.ctrl.get_quizzes()
+        # Since there is no quiz, the assertEquals function below fails
+        self.assertEquals(len(quizzes), 1, "There is no quiz.")
         
 
+    def test_expose_failure_02(self):
+        """
+        This test case aims to expose a crash when attempting to add a question to a non-existent id.
+        Expected crash: quizzes_controller.py, line 76
+        """
+        self.ctrl.clear_data()
```

Comment:
We need to relook at Test Case 2 as discussed.

Reply:
I have rechecked case 2 and made necessary changes.